<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photo_2022-05-28_17-57-46.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>590</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;
rgb(0, 255, 0)
</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>590</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Сгенерировать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>721</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/photo_2022-05-28_17-57-46.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/newPrefix/82.jpg&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>791</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: rgb(255, 255, 0);
color: rgb(186, 255, 178);
background-color: rgba(0, 0, 0, 80);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Пароль успешно добавлен&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#1f1f1f;&quot;&gt;Пароль должен состоять минимум из 4 символов&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 239, 239);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#55ffff;&quot;&gt;Пароль&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#1a1a1a;&quot;&gt;Ocean of keys&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; color:#55ffff;&quot;&gt;Соц. Сеть&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#07e8e8;&quot;&gt;Ocean of keys&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources>
  <include location="1234.qrc"/>
 </resources>
 <connections/>
</ui>
